app/admin/listing/[listingId]/insights/page.tsx

```diff
@@ -22,7 +31,7 @@ export default function Insights({ params }: { params: { listingId: string } })
     linkedin: [],
     website: [],
   });
-  
+  console.log(applicantData)
```

Comment:
Just a good idea to remove general data-related console logs!

Reply:
valid, forgot i added lol